scripts/apps/monitoring/directives/ItemActionsMenu.ts

```diff
@@ -17,6 +17,7 @@ interface IScope extends ng.IScope {
     item?: IArticle;
     open: any;
     active: any;
+    isRelatedItem: boolean;
```

Comment:
Can you make the directive take an optional `allowedActions: Array<string>` parameter so it's not coupled to any particular feature?